### DISCUSSION TITLE
Page breaks with merge cells in table

### DISCUSSION THREAD

**Discussion by lucasjdevine** [gotenberg/gotenberg#378] [Q&A]
I am creating tables in HTML and whenever there is a merged cell or column that goes across page breaks, it looks wonky. Is there something I can do to clean those columns and/or rows up across pages?

**Comment by gulien** [comment]
Hello @lucasjdevine,

You could make it work with some CSS. For instance: https://www.geeksforgeeks.org/how-to-apply-css-page-break-to-print-a-table-with-lots-of-rows/amp/

**Reply by lucasjdevine** [reply]
Normally, yes. And now granted, these tables I am working with are from clients and are ugly and long. No matter what CSS page break goes in to them, the merge cell issue remains the same. The pdf version of the document ends up with the continuing cell all the way down a page, while the cells that are small get broken, say 3/4 down the page. So the issue is the gap on the bottom 1/4 of the page where one cell continues on.

**Reply by gulien** [reply]
😄 HTML table hell! 

If that’s possible, you could play with the page size / margins to test if it helps with your issue (spoiler: I don’t think it does).

You may also check if this issue has been mentioned by puppeteer’s users (Gotenberg and Puppeteer use the same protocol).
